google/cloud/bigtable/client_options_test.cc

```diff
@@ -380,34 +356,36 @@ TEST(ClientOptionsTest, SetServiceConfigJSON) {
   bigtable::ClientOptions client_options_object = bigtable::ClientOptions();
   client_options_object.SetServiceConfigJSON("test-config");
   grpc::ChannelArguments c_args = client_options_object.channel_arguments();
-  auto const actual = GetString(client_options_object.channel_arguments(),
-                                GRPC_ARG_SERVICE_CONFIG);
+  auto const actual = ::google::cloud::internal::GetStringChannelArgument(
+      client_options_object.channel_arguments(), GRPC_ARG_SERVICE_CONFIG);
   ASSERT_TRUE(actual.has_value());
   EXPECT_EQ(*actual, "test-config");
 }
 
 TEST(ClientOptionsTest, SetUserAgentPrefix) {
   bigtable::ClientOptions client_options_object = bigtable::ClientOptions();
   client_options_object.SetUserAgentPrefix("test_prefix");
-  auto const actual = GetString(client_options_object.channel_arguments(),
-                                GRPC_ARG_PRIMARY_USER_AGENT_STRING);
+  auto const actual = ::google::cloud::internal::GetStringChannelArgument(
+      client_options_object.channel_arguments(),
+      GRPC_ARG_PRIMARY_USER_AGENT_STRING);
   ASSERT_TRUE(actual.has_value());
-  EXPECT_THAT(*actual, HasSubstr("test_prefix"));
+  EXPECT_THAT(*actual, ::testing::HasSubstr("test_prefix"));
```

Comment:
Nit: The `using ::testing::HasSubstr` was probably a feature.

Reply:
Reverted.